modules/wmco-prerequisites.adoc

```diff
@@ -7,8 +7,37 @@
 
 The following information details the supported platform versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
 
-[id="wmco-prerequisites-supported-6.0.0_{context}"]
-== WMCO 7 supported platforms and Windows Server versions
+[id="wmco-prerequisites-supported-7.2.0_{context}"]
+== WMCO 7.2.x supported platforms and Windows Server versions
```

Comment:
Do you plan to leave this as a letter variable since it applies to 7.2.0 and 7.2.1? If so, might suggest `7.2.z` instead of `7.2.x`